include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -5174,7 +5178,7 @@ namespace internal
     {
       Assert(fe_degree > -1, ExcInternalError());
       Assert(fe_eval.get_shape_info().element_type <=
-               MatrixFreeFunctions::tensor_symmetric,
+               MatrixFreeFunctions::tensor_symmetric_no_collocation,
```

Comment:
Here, I think we also need to play safe and only enable it for the `tensor_symmetric` case.